assets/src/scripts/components/hydrograph/time-series.js

```diff
@@ -191,11 +191,15 @@ export const secondaryYLabelSelector = createSelector(
 
 
 /**
- * @return {String}     The name of the currently selected variable.
+ * @return {String}     The title to include in the hyrdograph, will include method description if defined.
  */
 export const titleSelector = createSelector(
     getCurrentVariable,
-    variable => variable ? variable.variableName : ''
+    getCurrentMethodID,
+    getMethods,
+    (variable, methodId, methods) => variable ? variable.variableName +
```

Comment:
Please use string literals to format strings: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String. I also think this would be easier to read if you make this more than a one line function, finding the variable name part, then the method name part and the combining them. I like concise code but not at the expense of understanding.